tests/benchmark/bm_initialization/bm_basics_initialize_int8.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+/**************************************
+  Define and register tests
+  NOTE: benchmarks' tests order can affect their results. Please add new benchmarks at the end of
+the file.
+***************************************/
+#define REGISTER_Range_INT8_HNSW(BM_FUNC)                                                          \
+    BENCHMARK_REGISTER_F(BM_VecSimBasics, BM_FUNC)                                                 \
+        ->Args({50, 1})                                                                            \
+        ->Args({50, 10})                                                                           \
+        ->Args({50, 100})                                                                          \
+        ->Args({65, 1})                                                                            \
+        ->Args({65, 10})                                                                           \
+        ->Args({65, 100})                                                                          \
+        ->Args({80, 1})                                                                            \
+        ->Args({80, 10})                                                                           \
+        ->Args({80, 100})                                                                          \
+        ->ArgNames({"radiusX100", "epsilonX1000"})                                                 \
+        ->Iterations(10)                                                                           \
+        ->Unit(benchmark::kMillisecond)
+// Memory BF
```

Comment:
Why use different args for benchmarking range with int8 and not the common ones we use for all other types?

Reply:
In the dataset I used (wikipedia-int8), the previous range query  resulted in no result (i.e no vectors in X range from query vector), so I used bigger range to fit the current dataset.
The range query results were also validated with a python script.

Reply:
I see. Perhaps we can make these arguments our common ones for all types in range search to reduce complexity. WDYT @meiravgri ?

Reply:
IMO We should not touch the current benchmarks values.
We can change the range definition macro to receive additional arguments that will determine the range query parameters.
something like:
```
#define REGISTER_Range(BM_FUNC, radius, epsilons)
->ArgsRange(combine(radiuses, epsilons))
```
I believe google benchmark suggests a suitable API to achieve values combinations.

Reply:
For now a better location for this defintion would be in bm_basics_single_int8.cpp

Reply:
Why we didn't have to change this for brute force as well? Worth checking if we even get any results there